src/test/java/pl/edu/agh/iet/katabank/steps/DepositManagementSteps.java

```diff
@@ -35,8 +35,10 @@
     private Set<Deposit> customerDeposits;
     private LocalDate date;
     private BigDecimal amount;
+    private BigDecimal depositedAmount;
     private InterestPolicy interestPolicy;
     private DepositDurationDetails durationDetails;
+    LocalDate withdrawalDate;
```

Comment:
Since it's used only in one method, use local variable instead

Reply:
Right. I used in another test previously. Now it should be local.
Corrected.